Tests/Unit/Result/ConverterTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of the ONGR package.
+ *
+ * (c) NFQ Technologies UAB <info@nfq.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ONGR\ElasticsearchBundle\Tests\Unit\Result;
+
+use ONGR\ElasticsearchBundle\Result\Converter;
+
+/**
+ * Tests result converter.
+ *
+ * Class ConverterTest
+ *
+ * @package ONGR\ElasticsearchBundle\Tests\Unit\Result
+ */
+class ConverterTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * Tests if ConvertToDocument() throws Exception.
+     *
+     * @expectedException        \LogicException
+     * @expectedExceptionMessage Got document of unknown type 'foo'.
+     */
+    public function testConvertToDocumentException()
+    {
+        $converter = new Converter([], []);
+        $converter->convertToDocument(['_type' => 'foo']);
+    }
+
+    /**
+     * Tests assignArrayToObject().
+     */
+    public function testAssignArrayToObject()
+    {
+        $stub = new \StdClass();
```

Comment:
This should be at least document class mock with setter stubs. We want to test if converter actually calls setters.